### PR TITLE
[Jsontracer] Add index.html for Jsontracer webview

### PR DESCRIPTION
**Pull request by wnsdud4197**
this html is base for Jsontracer webview. later this html will be imported
ONE-vscode-DCO-1.0-Signed-off-by: Junyeong Yang <wnsdud4197@naver.com>

### CODE REVIEW

**Review by seanshpark — DISMISSED** [review]

File: media/Jsontracer/index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
```
Comment (llFreetimell) [inline media/Jsontracer/index.html:3]: It may not be possible to insert this at the top of file, am I right?

Comment (wnsdud4197) [reply media/Jsontracer/index.html:3]: [here](https://github.com/catapult-project/catapult/blob/444aba89e1c30edf348c611a9df79e2376178ba8/tracing/tracing_examples/trace_viewer.html), put it under the html tag to mark the license in the browser.

**Review by llFreetimell — DISMISSED** [review]
LGTM

**Review by parkjisu6239 — DISMISSED** [review]

**Review by seanshpark — APPROVED** [review]

**Comment by seanshpark** [comment]
Your signed off message is like
```
ONE-vscode-DCO-1.0-Signed-off-by: wnsdud4197 wnsdud4197@naver.com
```
which would be fixed to
```
ONE-vscode-DCO-1.0-Signed-off-by: wnsdud4197@naver.com <wnsdud4197@naver.com>

File: media/Jsontracer/index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        </header>
+        <section>
+          <div class="selected"></div>
+        </section>
+      </article>
+    </main>
+  </div>
+  <script nonce="nonce" type="module" src="scriptUri"></script>
+</body>
+</html>
```
Comment (seanshpark) [inline media/Jsontracer/index.html:89]: Please revise commit message

**Review by seanshpark — CHANGES_REQUESTED** [review]
Please revise commit message

**Comment by llFreetimell** [comment]
Little bit more precisely, you should insert your real name like following :)

`ONE-vscode-DCO-1.0-Signed-off-by: YOUR_REAL_NAME <wnsdud4197@naver.com>`

**Review by seanshpark — APPROVED** [review]

**Review by parkjisu6239 — APPROVED** [review]

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**